packages/backend/src/clients/Slack/SlackMessageBlocks.ts

```diff
@@ -309,3 +309,32 @@ export const getUnfurlBlocks = (
                   }),
     },
 });
+
+export const getNotificationChannelErrorBlocks = (
+    schedulerName: string,
+    error: any,
+): KnownBlock[] =>
+    getBlocks([
```

Comment:
Would be nice to have a link to the scheduler logs.

Reply:
sure, I think this can be done separately if that's ok